compiler/src/iree/compiler/ConstEval/Runtime.cpp

```diff
@@ -22,17 +23,8 @@ namespace {
 LogicalResult handleRuntimeError(Location loc, iree_status_t status) {
   if (iree_status_is_ok(status))
     return success();
-  std::string message;
-  message.resize(512);
-  iree_host_size_t buffer_length;
-  if (!iree_status_format(status, message.size(), &message[0],
-                          &buffer_length)) {
-    message.resize(buffer_length + 1);
-    iree_status_format(status, message.size(), &message[0], &buffer_length);
-  }
-  message.resize(buffer_length);
-  iree_status_ignore(status);
-  return emitError(loc) << "runtime error in consteval: " << message;
+  return emitError(loc) << "runtime error in consteval: "
+                        << iree::Status::ToString(status);
```

Comment:
FYI there was a memory issue in here that was crashing. The `iree::Status::ToString` helper is fine though.